packages/patternfly-4/react-table/src/components/Table/RowWrapper.tsx

```diff
@@ -64,7 +88,7 @@ class RowWrapper extends React.Component {
     return (
       <tr
         {...props}
-        ref={trRef}
+        ref={trRef as React.Ref<any>}
```

Comment:
It will work and should not complain to the consumer since it eventually resolves to an element. This is just bypassing the static typechecking I guess. FWIW, if all Refs are passed as React.Ref type, TypeScript will not complain, otherwise it seems we'll have to cast to support `Function`.